core/src/utilities/helpers/routing-helpers.js

```diff
@@ -597,6 +597,30 @@ class RoutingHelpersClass {
     };
     component.showAlert(alertSettings, false);
   }
+
+  // Adds and remove properties from searchParams
+  modifySearchParam(params, searchParams, paramPrefix) {
+    for (const [key, value] of Object.entries(params)) {
+      const paramKey = paramPrefix ? `${paramPrefix}${key}` : key;
+
+      searchParams.set(paramKey, value);
+      if (value === undefined) {
+        searchParams.delete(paramKey);//TODO change to key
```

Comment:
Do we need to change back to Key?  https://github.com/SAP/luigi/pull/2472/files#diff-60b3f2d3aa92a9cc5e51872f739096e2023d2a5412d5b6c0d566c73458186463L76

Reply:
No, not needed. Was a comment for me to avoid merge conflicts